src/Tests/Microsoft.NET.Perf.Tests/PerfTest.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Microsoft.Xunit.Performance.Api;
+using Xunit;
+
+//  Don't run any perf tests in parallel
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
+
+namespace Microsoft.NET.Perf.Tests
+{
+    public class PerfTest
```

Comment:
I thought that both perf frameworks provided infrastructure for all this. Why are we writing our own?